### PR TITLE
Rename store keyword

### PR DESCRIPTION
**Pull request by ruks**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Follow the instructions below to deploy WSO2 API Manager (WSO2 API-M) in a distr
     The following steps describe how to download, install, and configure WSO2 API Manager, with five instances.
     
     1.  Download the [WSO2 API Manager](http://wso2.com/products/api-manager/) in each of the five servers in the cluster for distributed deployment.
-    2.  Unzip the WSO2 API Manager zipped archive, and rename each of those directories respectively as Key Manager, Gateway, Publisher, Store, and Traffic Manager.
+    2.  Unzip the WSO2 API Manager zipped archive, and rename each of those directories respectively as Key Manager, Gateway, Publisher, Dev Portal, and Traffic Manager.
```
Comment (kavishkafernando) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Shouldn't it be **Developer Portal** for the descriptions?

Comment (npamudika) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Shall we change "Dev Portal" to "Developer Portal" as we have used the same in the other pages?

Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Fixed

Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Fixed

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 2]
```diff
@@ -579,14 +579,14 @@ This section involves setting up the API Store node and enabling it to work with
...
 
     !!! warning
-        This is **ONLY applicable** if you need to configure **HA for the Store.**
+        This is **ONLY applicable** if you need to configure **HA for the Dev Portal.**
 
-    Make a copy of the active instance configured above and use this copy as the second API Store active instance.
+    Make a copy of the active instance configured above and use this copy as the second Developer Portalactive instance.
 
-4.  Start the API Store node(s) by typing the following command in the command prompt. For more information on starting a WSO2 server, see [Starting the server](../../../InstallationGuide/running-the-product/) .
+4.  Start the Developer Portalnode(s) by typing the following command in the command prompt. For more information on starting a WSO2 server, see [Starting the server](../../../InstallationGuide/running-the-product/) .
```
Comment (npamudika) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: A space is mising

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 3]
```diff
@@ -579,14 +579,14 @@ This section involves setting up the API Store node and enabling it to work with
...
 
-3.  Optionally, configure High Availability (HA) for the Store.
+3.  Optionally, configure High Availability (HA) for the Dev Portal.
 
     !!! warning
-        This is **ONLY applicable** if you need to configure **HA for the Store.**
+        This is **ONLY applicable** if you need to configure **HA for the Dev Portal.**
 
-    Make a copy of the active instance configured above and use this copy as the second API Store active instance.
+    Make a copy of the active instance configured above and use this copy as the second Developer Portalactive instance.
```
Comment (npamudika) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: missing space in Portalactive

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 4]
```diff
@@ -579,14 +579,14 @@ This section involves setting up the API Store node and enabling it to work with
             revoke_endpoint = "https://[API-Gateway-LB-Host-or-IP]:${https.nio.port}/revoke"
         ```
 
-3.  Optionally, configure High Availability (HA) for the Store.
+3.  Optionally, configure High Availability (HA) for the Dev Portal.
 
     !!! warning
-        This is **ONLY applicable** if you need to configure **HA for the Store.**
+        This is **ONLY applicable** if you need to configure **HA for the Dev Portal.**
```
Comment (npamudika) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Shall we change Dev Portal to Developer Portal in all the places in descriptions?

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 2 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Fixed

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 3 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Fixed

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md (hunk 4 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md]: Fixed

**Pull request merged**